external-crates/move/crates/move-unit-test/tests/test_sources/rand_test.exp

```diff
@@ -0,0 +1,55 @@
+Running Move unit tests
+[ FAIL    ] 0x1::rand_test::should_fail_test_div_mod_10
+[ FAIL    ] 0x1::rand_test::should_fail_test_div_mod_10_2_vec
+[ FAIL    ] 0x1::rand_test::should_fail_test_expected_failure
+[ PASS    ] 0x1::rand_test::should_pass_test_expected_failure_pass
+[ TIMEOUT ] 0x1::rand_test::should_timeout_test_timeout
+
+Test failures:
+
+Failures in 0x1::rand_test:
+
+┌── should_fail_test_div_mod_10 ────── (seed = 8)
+│ error[E11001]: test failure
+│   ┌─ rand_test.move:4:11
+│   │
+│ 3 │     fun should_fail_test_div_mod_10(x: u64) {
+│   │         --------------------------- In this function in 0x1::rand_test
+│ 4 │         x / (x % 10);
+│   │           ^ Test was not expected to error, but it gave an arithmetic error originating in the module 0000000000000000000000000000000000000000000000000000000000000001::rand_test rooted here
+│ 
+│ 
+│ This test uses randomly generated inputs. Rerun with `.. move test should_fail_test_div_mod_10 --seed 8` to recreate this test failure.
+│ 
+└──────────────────
+
+
+┌── should_fail_test_div_mod_10_2_vec ────── (seed = 5)
+│ error[E11001]: test failure
+│   ┌─ rand_test.move:9:33
+│   │
+│ 8 │     fun should_fail_test_div_mod_10_2_vec(x: vector<vector<u64>>) {
+│   │         --------------------------------- In this function in 0x1::rand_test
+│ 9 │         std::vector::length(&x) / (std::vector::length(&x) % 10);
+│   │                                 ^ Test was not expected to error, but it gave an arithmetic error originating in the module 0000000000000000000000000000000000000000000000000000000000000001::rand_test rooted here
```

Comment:
Not from this PR, but I still find it super weird that in the same message we print out both 
`0x1::rand_test` and `0000000000000000000000000000000000000000000000000000000000000001::rand_test`

Worth fixing IMO

Reply:
Totally agreed, and shouldn't be hard. I'll put up a separate PR to fix this just for cleanliness sake.